### PR TITLE
Feature readme

### PR DESCRIPTION
**Pull request by cathsu**
Creating a new pull request: merge feature-readme with master branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
 Light & Motion project. Purpose: To read in a CSV data file and output its raw data and graphic visualization into an Excel file. Currently only works with an Lumensphere CSV file. 
+
+* Derived Data Imjin 800.csv- raw CSV file
```
Comment (cathsu) [inline README.md:3]: Trying to figure out how reviewing works.

**Review by JosephMatos — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (JosephMatos) [reply README.md:3]: Seems like a good opportunity for us to mess around with the features, this is great!

**Review by JosephMatos — APPROVED** [review]
Commenting and approving.

**Review by JosephMatos — APPROVED** [review]
I think this is a great start to documentation that explains the functionality of the program. Along with in-text documentation for the purpose of specific algorithms/data types. Awesome!

**Review by JosephMatos — APPROVED** [review]

**Pull request merged**